### ISSUE TITLE
DW texts: Minor INT3-issue

### ISSUE THREAD

**Issue by BOberreither** [semantic-kraus/dw-data#40]
### INT3 statements in DW texts 
are mostly great. One thing came up:
If a ref type int refers to a citedRange within a bibl that contains **another** citedRange that is wholeText, the referred-to-URI must be built from that wholeText citedRange xml:id.
Currently:
```
#INT3 intertext relationship
<https://sk.acdh.oeaw.ac.at/DWbibl00070/relation/0> a ns1:INT3_IntertextualRelationship  ;
  rdfs:label "Intertextual relation"@en ;
  ns1:R12_has_referred_to_entity <https://sk.acdh.oeaw.ac.at/DWbibl01964/passage/1> ;
  ns1:R13_has_referring_entity <https://sk.acdh.oeaw.ac.at/DWbibl00070/passage/0> .
```
Line 3 of the statement should be 
```
ns1:R12_has_referred_to_entity <https://sk.acdh.oeaw.ac.at/DWbibl01965/passage/1> ;
```

**Comment by BergListe** [comment]
Done.

**Comment by BOberreither** [comment]
@BergListe Well, you did even more - added another ID to the URI, line 3 is now: 
`ns1:R12_has_referred_to_entity <https://sk.acdh.oeaw.ac.at/DWbibl01965 DWbibl01335/passage/1> ;`
I think this is the issue I mentioned in https://github.com/semantic-kraus/dw-data/issues/41 – it seems like a condition (@wholeText) is missing, so it just takes all the IDs in all the citedRange elements.

**Comment by BergListe** [comment]
This should be fixed - see #41
